ui/src/components/chat/sidebar.rs

```diff
@@ -510,8 +512,7 @@ pub fn Sidebar(cx: Scope<Props>) -> Element {
                                 with_badge: badge,
                                 onpress: move |_| {
                                     state.write().mutate(Action::ChatWith(&chat_with.id, false));
-
-                                    if state.read().ui.is_minimal_view() {
```

Comment:
`is_minimal_view` is used in a lot of places. if it doesn't work, it may be better to calculate `WindowMeta` and use `Action::SetMeta` to update state so that `is_minimal_view` works correctly.

Reply:
I think you're right, prob going to need a state handler to calculate everything and rely on that